pkg/stis/calstis/cs2/wscript

```diff
@@ -5,7 +5,7 @@ def build(bld):
         name = 'cs2.e',
         source = 'cs2.c',
         target = 'cs2.e',
-        use = ['calstis', 'imphttab'] + bld.env.LOCAL_LIBS,
+        use = ['hstcallib', 'calstis', 'imphttab'] + bld.env.LOCAL_LIBS,
         lib = bld.env.EXTERNAL_LIBS,
```

Comment:
Is hstcallib necessary here?

Reply:
No, because cs2.c doesn't use the PtrRegister. I probably added this when adding the rest as a blind list through the main files. I'll remove,  thanks.